<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaoShiManage</class>
 <widget class="QWidget" name="KaoShiManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KaoShiManage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>科目：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>请输入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>班级：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="placeholderText">
          <string>请输入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="query">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>任务名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>科目</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>人员</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>预计考试时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>实际开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>实际结束时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>计算机打分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>教师打分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="acStart">
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
  </action>
  <action name="acPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="acStop">
   <property name="text">
    <string>终止</string>
   </property>
   <property name="toolTip">
    <string>终止</string>
   </property>
  </action>
  <action name="acDetail">
   <property name="text">
    <string>查看详情</string>
   </property>
  </action>
  <action name="acDelTask">
   <property name="text">
    <string>删除任务</string>
   </property>
  </action>
  <action name="acEditTask">
   <property name="text">
    <string>编辑任务</string>
   </property>
  </action>
  <action name="acReplay">
   <property name="text">
    <string>视频回放</string>
   </property>
   <property name="toolTip">
    <string>视频回放</string>
   </property>
  </action>
  <action name="acTeacherScore">
   <property name="text">
    <string>教师打分</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
